subprojects/core/src/main/java/org/gradle/api/internal/tasks/execution/SkipUpToDateTaskExecuter.java

```diff
@@ -65,7 +65,7 @@ public void execute(TaskInternal task, TaskStateInternal state, TaskExecutionCon
     private void logOutOfDateMessages(List<String> messages, TaskInternal task, String took) {
         if (LOGGER.isInfoEnabled()) {
             Formatter formatter = new Formatter();
-            formatter.format("Executing %s (up-to-date check took %s) due to:", task, took);
+            formatter.format("Up-to-date check for %s took %s. It is not up-to-date because:", task, took);
```

Comment:
I guess we should also log the time it took for up-to-date check when the task was up-to-date.

Reply:
Like here: https://github.com/gradle/gradle/pull/3111/files#diff-2125ea3ea2f91ebdc25681fe80e832ebR124 ?

Reply:
:+1: